gateway/src/cluster/builder.rs

```diff
@@ -56,6 +56,21 @@ pub enum ShardScheme {
     /// For example, if Discord recommends 10 shards, then all 10 shards will be
     /// started.
     Auto,
+    /// Star a single bucket for very large bot sharding
+    Bucket {
+        /// The shard id of the first shard to start, this should be less than the concurrency.
```

Comment:
```suggestion
        /// The ID of the first shard to start.
        ///
        /// This should be less than the concurrency.
```
and then the rest. First lines should be concise since they're summaries